### DISCUSSION TITLE
Complaint

### DISCUSSION THREAD

**Discussion by Lizzgg** [community/community#30956] [Other Feature Feedback, Questions, & Ideas] [answered]
Please I misplaced my token please how can I get it back please

**Comment by airtower-luna** [comment, marked as answer]
You can't, but you can revoke it and [create a new one](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) at any time. I hope that helps! :smiley_cat:

**Comment by Lizzgg** [comment]
If I revoke it won't I loose my data in GitHub

**Comment by airtower-luna** [comment]
No. Revoking a personal access token just makes whatever used that token not have access any more. Considering you lost the token already that's no harm.

What would make you lose your data is deleting your *account*, or losing the login data.
